implementations/rust/ockam/ockam_app/linux-desktop-template

```diff
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Categories={{categories}}
+{{#if comment}}
+Comment={{comment}}
+{{/if}}
+Exec={{exec}} %U
+Icon={{icon}}
+Name={{name}}
+Terminal=false
+Type=Application
+MimeType=x-scheme-handler/ockam;
+X-KDE-Protocols=ockam;
```

Comment:
For my own curiosity - are these two sufficient to also cover Gnome/XFCE?

Reply:
Yes, `MimeType=x-scheme-handler/ockam;` is a Freedesktop convention, and Debian by default should have an automatic hook that calls `update-desktop-database`